RandomImg/random.js

```diff
@@ -0,0 +1,12 @@
+
+const Random = document.querySelector('#randomize');
+const RandomImg = function () {
+    console.log('making a tweet...');
+
+    const newImg = document.createElement('img');
+    newImg.setAttribute('src', 'https://source.unsplash.com/random');
```

Comment:
you can make an API call with different sized images by getting the height and width value from the DOM and assigning them to a variable, then using those variables as part of the URL

similar to this
```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```